### PR TITLE
Make some arrays const.

### PR DESCRIPTION
**Pull request by pfusik**
While reading cc65 sources I noticed variable arrays that could be const.

### CODE REVIEW

File: src/cc65/codegen.c [hunk 1]
```diff
@@ -3546,7 +3546,7 @@ void g_ne (unsigned flags, unsigned long val)
 void g_lt (unsigned flags, unsigned long val)
 /* Test for less than */
 {
-    static const char* ops[12] = {
+    static const char* const ops[4] = {
         "tosltax", "tosultax", "toslteax", "tosulteax",
```
Comment (oliverschmidt) [inline src/cc65/codegen.c]: I guess you want the final ',' removed here too.

File: src/grc65/main.c [hunk 2]
```diff
@@ -226,18 +226,15 @@ static void openSFile (void)
...
-static int findToken (const char **tokenTbl, const char *token)
+static int findToken (const char * const *tokenTbl, const char *token)
 {
     /* takes as input table of tokens and token, returns position in table or -1 if not found */
-    int a = 0;
-
-    while (strlen (tokenTbl[a]) != 0) {
-        if (strcmp (tokenTbl[a], token) == 0) break;
-        a++;
+    int i;
```
Comment (oliverschmidt) [inline src/grc65/main.c]: Please keep the blank line separating the definitions from the code.

File: src/grc65/main.c [hunk 3]
```diff
@@ -226,18 +226,15 @@ static void openSFile (void)
...
+    for (i = 0; tokenTbl[i][0]; i++) {
+        if (strcmp (tokenTbl[i], token) == 0) return i;
```
Comment (oliverschmidt) [inline src/grc65/main.c]: Please put 'return' into own {} block (and own line).

File: src/grc65/main.c [hunk 4]
```diff
@@ -226,18 +226,15 @@ static void openSFile (void)
...
     }
 
-    if (strlen (tokenTbl[a]) == 0) a = -1;
-    return a;
+    return -1;
 }
```
Comment (oliverschmidt) [inline src/grc65/main.c]: Please elaborate why you changed the code above beyond the scope of this pull request to add 'const'.

**Comment by pfusik** [comment]
Added a commit with the suggested edits.

I changed a parameter type of `findToken` and I found this short function overcomplicated, so refactored it to be easier to read.

**Comment by oliverschmidt** [comment]
Thanks for the contribution - and the explanation :-)

**Pull request merged**